include/revng/ABI/FunctionType-revng-c.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
```

Comment:
Let's keep it in revng for now, we'll move it later.
Can you name these files `ConvertToCABI` and `ConvertToRaw`?

Reply:
Why though? Consider that this PR already has a sister revng-c MR since the analysis is post-DLA and needs to access the kinds there.

I feel like now's the time to cut, we just need to think of a good place to put in on the "other side", it would feel somewhat sad to introduce `revngcABI.so` as well.

Reply:
I'm confused, we definitely do not want a file with `revng-c` in the name.
Should I review it here or in `revng-c`?

Reply:
Give it a round of reviews here ignoring the name. And if everything is more or less fine, I'll move it over there.

Reply:
(the reason I prefer to do it this way is because I expect a big deal of discussion to be shared between both directions, so it's easier having them side by side for now. And after we're more or less on the same page on how they should look, it's easy to finally cut them apart).